consensus/src/main/java/org/apache/iotdb/consensus/ratis/metrics/IoTDBMetricRegistry.java

```diff
@@ -74,6 +74,16 @@ public class IoTDBMetricRegistry implements RatisMetricRegistry {
   public static final String FOLLOWER_APPEND_ENTRIES_LATENCY = "follower_append_entry_latency";
   /** Time taken to process write requests from client. */
   public static final String RAFT_CLIENT_WRITE_REQUEST = "clientWriteRequest";
+  /** Applied index of state machine */
+  public static final String STATEMACHINE_APPLIED_INDEX_GAUGE = "appliedIndex";
+  /** Last log index which completely applied to the state machine */
+  public static final String STATEMACHINE_APPLY_COMPLETED_GAUGE = "applyCompletedIndex";
+  /** Next index of peer */
```

Comment:
Is these metrics not used yet?

Reply:
These metrics are already used in the RatisConsensus Metrics. Maybe we can put these changes in another PR.